app/controllers/projects/samples_controller.rb

```diff
@@ -29,13 +29,12 @@ def edit; end
     def create
       @sample = Samples::CreateService.new(current_user, @project, sample_params).execute
 
-      respond_to do |format|
-        if @sample.save
-          flash[:success] = t('.success')
-          format.html { redirect_to namespace_project_sample_path(id: @sample.id) }
-        else
-          format.html { render :new, status: :unprocessable_entity }
-        end
+      if @sample.persisted?
+        flash[:success] = t('.success')
+        redirect_to namespace_project_sample_path(id: @sample.id)
+      else
+        flash[:error] = @sample.errors.full_messages.first
```

Comment:
This has the same behaviour as the comment above, so please remove the flash here.

Reply:
Updated in 2365691109fea5b67fd2bc7040892d5ad22edff2